pkg/registry/scheduling/rest/storage_scheduling.go

```diff
@@ -101,7 +101,7 @@ func AddSystemPriorityClasses() genericapiserver.PostStartHookFunc {
 			return true, nil
 		})
 		// if we're never able to make it through initialization, kill the API server.
-		if err != nil {
+		if err != nil && !apierrors.IsAlreadyExists(err) {
```

Comment:
[https://github.com/kubernetes/kubernetes/blob/6219e51283a75752f452b7a1fb7c1c3c076bf26f/pkg/registry/scheduling/rest/storage_scheduling.go#L86](https://github.com/kubernetes/kubernetes/blob/6219e51283a75752f452b7a1fb7c1c3c076bf26f/pkg/registry/scheduling/rest/storage_scheduling.go#L86)

i don't really follow but where does the `AlreadyExistsError` come from?

Reply:
> the AlreadyExists error should be tolerated on line 87, not here

agree. returning a `false, nil` here will give a retry, amiright?